pegasus/sites.v3/code.org/public/educate/weblab.haml

```diff
@@ -0,0 +1,20 @@
+---
+title: Web Lab
+theme: responsive
+video_player: true
+---
+
+%link{href: "/css/jumbotron-banner.css", rel: "stylesheet"}
+%link{href: "/shared/css/course-blocks.css", rel: "stylesheet"}
+
+= view :jumbotron_banner, img: CDO.code_org_url("/images/app-lab/girl-cropped.jpg"), title: "Web Lab (beta)", desc: "Web Lab is a programming environment where you can make simple web pages using HTML and CSS. Design your web pages and share your site in seconds.", specs: "Ages 13+, all modern browsers, English only", url: CDO.studio_url('/p/weblab'), cta: "Make an web page", photo_cred: "Photo courtesy of Fundación Sadosky", nav_bar: [{text: 'Resources', url: '#resources'}]
```

Comment:
good thought, I'm making a few pages that are all like this, and might do this as a follow up to all.